program-runtime/src/invoke_context.rs

```diff
@@ -185,13 +185,14 @@ pub struct SerializedAccountMetadata {
 pub struct InvokeContext<'a> {
     /// Information about the currently executing transaction.
     pub transaction_context: &'a mut TransactionContext,
+    /// The local program cache for the transaction batch.
```

Comment:
Why the reordering?

Reply:
I plan to evict the other one soon, so wanted to organize things a bit.